release_notes/41-release-notes.adoc

```diff
@@ -0,0 +1,425 @@
+
+[NOTE]
+====
+Standards controlled by the compliance operator cannot be disabled or hidden in {product-title-short}. You must configure the compliance operator custom resource definitions (CRDs) to remove them from display.
+====
+
+[id="declarative-configurations-for-authentication-and-authorization_{context}"]
+=== Declarative configurations for authentication and authorization
+
+Customers following the cloud-native paradigm, prefer to use declarative configurations for versioning and auditability. Declarative configurations allow you to use configuration management tools to configure and update resources in your environment, rather than configuring resources directly in the system each time you want to make changes.
+
+After using declarative configuration to specify parameters for resources, you can quickly update configurations that are stored in repositories and apply them to the system using a GitOps workflow.
+
+You can create declarative configurations for authentication and authorization resources such as authentication providers, roles, permission sets, and access scopes. Configurations are stored in YAML files and mounted during Central installation.
+For more information, see xref:../operating/manage-user-access/manage-role-based-access-control-3630.adoc#declarative-configuration-auth-resources_manage-role-based-access-control[Declarative configuration for authentication and authorization resources].
+
+[id="sso-configuration-using-the-roxctl-cli_{context}"]
+=== SSO configuration using the roxctl CLI
+
+In {product-title-short} 4.1, you can enhance Central's security and login capabilities with the new feature of using an authentication provider to authorize the `roxctl` CLI.
+This feature enables you to use the `roxctl` CLI and integrate with authentication systems such as OpenID Connect (OIDC) for a seamless single sign-on (SSO) experience without requiring additional tokens or passwords.
+Previously, the `roxctl` CLI only supported token and password-based workflows.
+
+For more information, see xref:../cli/getting-started-cli.adoc#configuring-an-authentication-provider-in-central_cli-getting-started[Configuring an authentication provider in Central].
+
+[id="new-collection-method-based-on-bpf-co-re-technology-preview_{context}"]
+=== New collection method based on BPF CO-RE (Technology Preview)
+
+With this release, you can test a new runtime collection method that no longer requires a precompiled driver (either downloaded or provided through support package upload) for the host operating system (OS).
+
+:FeatureName: The `CORE_BPF` collection method
+include::snippets/technology-preview.adoc[leveloffset=+1]
+
+This feature is available for testing on the x86_64 architecture and is based on the new Linux BPF features, including CO-RE (Compile Once-Run Everywhere). To enable it, set the value of the `collector.collectionMethod` parameter of your backed-up cluster to `CORE_BPF`.
```

Comment:
@agantony Please change to: 
This feature is available for testing on x86_64 and s390x architectures and....

cc: @robbycochran